scripts/linters/pylint_extensions.py

```diff
@@ -1921,6 +1922,74 @@ def process_tokens(self, tokens):
                         'single-line-pragma', line=line_num)
 
 
+class TypeIgnoreCommentChecker(checkers.BaseChecker):
+    """Custom pylint checker which checks if MyPy's type ignores are properly
+    documented or not.
+    """
+
+    __implements__ = interfaces.ITokenChecker
+
+    ALLOWED_LINES_OF_GAP = 10
```

Comment:
> Should we keep this in feconfig?

I think `ALLOWED_LINES_OF_GAP`  is related to only this file, so instead of transferring it to the feconf I put it on the top of the file PTAL!